certora/specs/Health.spec

```diff
@@ -75,6 +78,44 @@ filtered {
     assert !priceChanged => stillHealthy;
 }
 
+// The general case of liquidate for the stayHealthy rule
+//   - the user checked is not the borrower,
+//   - the market of the liquidation is the market of the user,
```

Comment:
it times out without this?

Reply:
No I think we managed to prove it, but I did not feel like it was worth to add it (distinct storage touched). But we can add it if you think it makes sense

Reply:
you think that removing the assumption doesn't help understanding? Otherwise removing assumptions is always better no?

Reply:
I agree that less assumptions is always better but here we need the assumption to help the prover. The other part (where the market of the liquidation is not the market of the user) has been proved separately, but I did not add it as another rule, because I felt that it was not worth to add that rule. Do you think that we should add it too ?

Reply:
From our previous conversation actually I remember that you were in favor of adding it, I'll do it and then re-request a review :)